content/tutorial-build-run-rippled-ubuntu.md

```diff
@@ -0,0 +1,165 @@
+# Build and Run `rippled` on Ubuntu
+
+`rippled` is the core peer-to-peer server that manages the XRP Ledger. A `rippled` server can connect to a network of peers, relay cryptographically signed transactions, and maintain a local copy of the complete shared global ledger.
+
+For an overview of `rippled`, see [Operating rippled Servers](tutorial-rippled-setup.html).
+
+Use these instructions to build a `rippled` binary file and run it as a stock `rippled` server on Ubuntu 15.04 or later. These instructions were tested on Ubuntu 16.04 LTS.
+
+For information about building `rippled` for other platforms, see [Builds](https://github.com/ripple/rippled/tree/develop/Builds) in the `rippled` GitHub repository.
+
+
+## System Requirements
+
+**_To build `rippled`:_**
+
+You need a **minimum** of 8GB of RAM.
+
+**_To run `rippled`:_**
+
+Meet these [system requirements](tutorial-rippled-setup.html#system-requirements).
+
+
+## 1. Build `rippled`
+
+These instructions use Ubuntu's APT (Advanced Packaging Tool) to install the software prerequisites you need to build and run `rippled`.
+
+1. Update the list of packages that are available for `apt-get` to install or upgrade.
+
+        sudo apt-get update
+
+2. Upgrade currently installed packages.
+
+        sudo apt-get -y upgrade
+
+3. Install Git.
+
+        sudo apt-get -y install git
+
+4. Install SCons.
+
+        sudo apt-get -y install scons
+
+5. Install Ctags.
+
+        sudo apt-get -y install ctags
+
+6. Install `pkg-config`.
+
+        sudo apt-get -y install pkg-config
+
+7. Install Protocol Buffers.
+
+        sudo apt-get -y install protobuf-compiler
+        sudo apt-get -y install libprotobuf-dev
+
+8. Install Secure Socket Layer (SSL) toolkit development files.
+
+        sudo apt-get -y install libssl-dev
+
+9. Install `python-software-properties`.
+
+        sudo apt-get install -y python-software-properties
+
+10. Install Boost.
+
+        sudo add-apt-repository -y ppa:boost-latest/ppa
+        sudo apt-get -y update
+        sudo apt-get install -y libboost-all-dev
+
+11. Get `rippled` source code.
+
+        git clone https://github.com/ripple/rippled.git
+        cd rippled
+        git checkout master
+
+12. Build `rippled` binary executable from source code. This may take about 30 minutes, depending on your hardware specs.
+
+        scons
+
+    SCons saves the built executable in `rippled/build`.
```

Comment:
Scons will likely be deprecated in favor of cmake in the near future

This would mean installing `cmake` instead of `scons` (Step 4)
The build step could be either 
```
BUILD=cmake CC=gcc TARGET=release.unity bin/ci/ubuntu/build-and-test.sh
```
or
```
mkdir -p build/gcc.release
cd build/gcc.release
cmake ../.. -Dtarget=gcc.release
cmake --build .
```
For the former, `rippled` will be in `build/gcc.release.unity/` and unittests will run automatically
For the latter, `rippled` will be in `build/gcc.release`

Reply:
https://github.com/ripple/rippled/blob/develop/bin/ci/ubuntu/build-and-test.sh is a bit of a mess that expects all kinds of environment variables to be maybe existing, I wouldn't recommend it as the documented build step. Even `Builds/Test.py [and some options]` already gets messy with cmake.

The latter version is still better, even though it still contains lots of "magic" (like the "target" having to match a certain subfolder or the target variable containing a definition which compiler should be used in which configuration...).

Reply:
Thank you for the heads up, @wilsonianb. Thank you for the detailed feedback, @MarkusTeufelberger. I've opened a ticket to capture these details -- we'll update the doc when scons is deprecated in favor of cmake.